packages/core/src/process/4-MutationTestExecutor.ts

```diff
@@ -76,6 +77,14 @@ export class MutationTestExecutor {
     };
   }
 
+  private executeIgnore(input$: Observable<MutantTestCoverage>) {
+    const [notIgnoredMutant$, ignoredMutant$] = partition(input$.pipe(shareReplay()), ({ mutant }) => mutant.ignoreReason === undefined);
```

Comment:
Are the `notIgnoredMutant$` and ignoredMutant$` observables arrays? If so, could you make their name plural to show this?

Reply:
No, it is an observable of mutants. It is kind of a convention to name them plural with a `$` instead of `s`